files/rollup.config.mjs

```diff
@@ -8,6 +8,12 @@ const addon = new Addon({
   destDir: 'dist',
 });
 
+const rootDirectory = dirname(fileURLToPath(import.meta.url));
```

Comment:
import.meta.dirname is in node 20.11+

Reply:
right, which is why I'm not using `import.meta.dirname`, and did re-create the `__dirname`, but I called it `rootDirectory` instead of the legacy `__dirname` var

Reply:
ah sorry 🙈 you said import.meta.dirname and that's what I read in the code 🫠 brains are weird